wetterdienst/core/timeseries/export.py

```diff
@@ -475,52 +421,34 @@ def to_target(self, target: str):
             Examples::
 
                 # Prepare
-                alias fetch='wetterdienst dwd observation values --station=1048,4411 --parameter=kl --resolution=daily --period=recent'
+                alias fetch='wetterdienst dwd observation values --station=1048,4411 --parameter=kl --resolution=daily
+                --period=recent'
 
                 # Acquire data.
                 fetch --target="sqlite:///dwd.sqlite?table=weather"
 
                 # Query data.
                 sqlite3 dwd.sqlite "SELECT * FROM weather;"
 
-            """  # noqa:E501
-
-            # Honour SQLite's SQLITE_MAX_VARIABLE_NUMBER, which defaults to 999
-            # for SQLite versions prior to 3.32.0 (2020-05-22),
-            # see https://www.sqlite.org/limits.html#max_variable_number.
-            chunksize = 5000
-            if target.startswith("sqlite://"):
-                import sqlite3
-
-                if sqlite3.sqlite_version_info < (3, 32, 0):
-                    chunksize = int(999 / len(self.df.columns))
-
+            """
             log.info("Writing to SQL database")
-            self.df.to_sql(
-                name=tablename,
-                con=target,
-                if_exists="replace",
-                index=False,
-                method="multi",
-                chunksize=chunksize,
-            )
+            self.df.write_database(table_name=tablename, connection_uri=target, if_exists="replace")
```

Comment:
Dito. `method` and `chunksize` are missing on the new interface. That will probably be slow, unless Polars will apply any other magic. I will be happy to learn more.